.release-version

```diff
@@ -1 +1 @@
-14.50.1
+14.51.0
```

Comment:
Should the version change from 14.50.1 to 14.51.0 be considered a minor version update, or should it instead be a subversion update to 14.50.2?

Reply:
@seenanair I think it's a feature update because it has code related to BGE stuff, and a new feature to include the Norm plate. I think the last digit is for patches in [SemVer](https://semver.org/).

Reply:
👍